docs/pages/index.tsx

```diff
@@ -0,0 +1,45 @@
+import type { NextPage } from 'next';
+import Head from 'next/head';
+import styles from '../styles/Home.module.css';
+
+const Home: NextPage = () => {
+  return (
+    <div className={styles.container}>
+      <Head>
+        <title>CooperTS Documentation</title>
```

Comment:
Since Documentation seems to be a sub-page, maybe this should just be "CooperTS" or "CooperTS - Home"?